<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCode</class>
 <widget class="QMainWindow" name="SmartCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCode</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="ltCentralWidget">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="hSpliter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="vSpliter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="lLeftBar">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="lCode">
         <item>
          <widget class="QComboBox" name="cbCurrentInstructionInfo"/>
         </item>
         <item>
          <widget class="QTabWidget" name="twOpenedFiles">
           <widget class="QWidget" name="tbOpenedFile1">
            <attribute name="title">
             <string>Current File</string>
            </attribute>
            <layout class="QVBoxLayout" name="ltOpenedFile1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QStackedWidget" name="bottomPanel">
       <widget class="QWidget" name="pLog"/>
       <widget class="QWidget" name="pOutput"/>
       <widget class="QWidget" name="pInterpreter"/>
       <widget class="QWidget" name="pTerminal"/>
       <widget class="QWidget" name="pFindReplace"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>$File</string>
    </property>
    <addaction name="aNewFile"/>
    <addaction name="aAddFile"/>
    <addaction name="aDeleteFile"/>
    <addaction name="separator"/>
    <addaction name="aSaveCurrentFile"/>
    <addaction name="aSaveAllProjectFiles"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>$Edit</string>
    </property>
    <addaction name="action1_2"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>$View</string>
    </property>
    <addaction name="action1_3"/>
   </widget>
   <widget class="QMenu" name="menu_Panel">
    <property name="title">
     <string>$Panel</string>
    </property>
    <addaction name="aPanelTerminal"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>$About</string>
    </property>
    <addaction name="action1_5"/>
   </widget>
   <widget class="QMenu" name="menu_Git">
    <property name="title">
     <string>$Git</string>
    </property>
    <addaction name="action1_6"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>$Project</string>
    </property>
    <addaction name="aStart"/>
    <addaction name="aStartDebuging"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_Panel"/>
   <addaction name="menu_Git"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="aNewFile"/>
   <addaction name="aSaveCurrentFile"/>
   <addaction name="aSaveAllProjectFiles"/>
   <addaction name="separator"/>
   <addaction name="aStart"/>
   <addaction name="aStartDebuging"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="aNewFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/new_file.png</normaloff>:/icons/resources/new_file.png</iconset>
   </property>
   <property name="text">
    <string>$NewFile</string>
   </property>
   <property name="toolTip">
    <string>Добавть новый файл в проект</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="aAddFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/add_file.png</normaloff>:/icons/resources/add_file.png</iconset>
   </property>
   <property name="text">
    <string>$AddFile</string>
   </property>
   <property name="toolTip">
    <string>Добавить файл к проекту</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="aDeleteFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/delete_file.png</normaloff>:/icons/resources/delete_file.png</iconset>
   </property>
   <property name="text">
    <string>$DeleteFile</string>
   </property>
   <property name="toolTip">
    <string>Удалить файл из проекта</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="aSaveCurrentFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/save.png</normaloff>:/icons/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>$SaveCurrentFile</string>
   </property>
   <property name="toolTip">
    <string>Сохранить редактируемый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="aSaveAllProjectFiles">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/save_project.png</normaloff>:/icons/resources/save_project.png</iconset>
   </property>
   <property name="text">
    <string>$SaveAllProjectFiles</string>
   </property>
   <property name="toolTip">
    <string>Сохранить все файлы в проекте</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+S</string>
   </property>
  </action>
  <action name="action1_6">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="aStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/start.png</normaloff>:/icons/resources/start.png</iconset>
   </property>
   <property name="text">
    <string>$Start</string>
   </property>
   <property name="toolTip">
    <string>Запустить проект</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="aStartDebuging">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/debug.png</normaloff>:/icons/resources/debug.png</iconset>
   </property>
   <property name="text">
    <string>$StartDebuging</string>
   </property>
   <property name="toolTip">
    <string>Запустить отладку</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+R</string>
   </property>
  </action>
  <action name="aPanelTerminal">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/terminal.png</normaloff>:/icons/resources/terminal.png</iconset>
   </property>
   <property name="text">
    <string>$Terminal</string>
   </property>
   <property name="toolTip">
    <string>Показать панель с терминалом</string>
   </property>
   <property name="shortcut">
    <string>Meta+Ctrl+T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
